lib/puppet/util/run_mode.rb

```diff
@@ -85,21 +85,21 @@ def code_dir
       end
 
       def var_dir
-        which_dir(File.join(windows_common_base("var")), "~/.puppet/var")
+        which_dir(File.join(windows_common_base("puppet/cache")), "~/.puppet/var")
```

Comment:
Yeah, I don't think either @joshcooper or I are in disagreement here that this seems a little peculiar.  

However, this is how settings are mapped on the Linux side of the fence and we're ultimately shooting for consistency (even if it doesn't seem to make total sense).

See the full documentation at
https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md#puppet-agent-nix

The relevant / important bits you can see here:

| Setting  | Linux  | Windows |
| :------------ |:---------------|:-----|
| :vardir      | /var/lib/puppet/cache | C:\ProgramData\PuppetLabs\puppet\cache |
| :logdir      | /var/log/puppetlabs        |  C:\ProgramData\PuppetLabs\puppet\var\log |
| :rundir | /var/run/puppetlabs        |    C:\ProgramData\PuppetLabs\puppet\var\run |